### PR TITLE
Nuxt3 implement po c for bootstrap vue next on nuxt3 branch css fix

### PR DESCRIPTION
**Pull request by LucasSaid005**
### 📚 Description

_Describe your changes in detail. Why is this change required? What problem does it solve?_

----

### ❓ Type of change

_What types of changes does your code introduce? Put an `x` in all the boxes that apply._

- [ ] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: assets/scss/_cookie.scss [hunk 1]
```diff
@@ -9,8 +9,12 @@
...
     margin: 0;
-    
+
     a {
         color: $body-bg-colored;
     }
+
+    @media only screen and (width <= 700px) {
+        right: 0;
+    }
```
Comment (PeterThrels) [inline assets/scss/_cookie.scss:19]: why was this added?

File: assets/scss/_nav.scss [hunk 2]
```diff
@@ -18,7 +18,7 @@
 }
 
 #main-header {
-    .navbar {
+    .navbar-light {
```
Comment (PeterThrels) [inline assets/scss/_nav.scss:21]: there's no more class .navbar ?

File: components/organisms/ProfileIdentitySelection.vue [hunk 3]
```diff
@@ -29,3 +29,9 @@
 <script lang="ts" setup>
 import RegistrationCard from '~/components/molecules/registration/RegistrationCard.vue'
 </script>
+
+<style lang="scss" scoped>
+.fs-2rem {
+    font-size: 2rem;
+}
```
Comment (PeterThrels) [inline components/organisms/ProfileIdentitySelection.vue:36]: if you find that you have to make more font sizes, why not just extend bootstrap utility like we're already doing in variables.scss

File: components/molecules/registration/RegistrationCard.vue [hunk 4]
```diff
@@ -24,3 +26,8 @@ const props = defineProps<{
     link: string
 }>()
 </script>
+<style>
+.fs-text {
+    font-size: 1.75rem;
+}
```
Comment (PeterThrels) [inline components/molecules/registration/RegistrationCard.vue:32]: if you find that you have to make more font sizes, why not just extend bootstrap utility like we're already doing in variables.scss

File: components/molecules/HeaderComponent.vue [hunk 5]
```diff
@@ -1,7 +1,7 @@
 <template>
     <div class="container-fluid px-0 mb-5 pb-5">
-        <b-navbar class="navbar-padding-adjust" fixed="top" type="light" variant="light">
-            <b-container class="d-flex justify-content-between">
+        <b-navbar class="navbar-padding-adjust" fixed="top" type="light">
```
Comment (PeterThrels) [inline components/molecules/HeaderComponent.vue:3]: shouldn't class .navbar-light work, why use type?

File: pages/profile/password/change.vue [hunk 6]
```diff
@@ -193,3 +193,8 @@ const submitChangePassword = async () => {
     }
 }
 </script>
+<style>
+.fs-cp {
+    font-size: 2.5rem;
+}
```
Comment (PeterThrels) [inline pages/profile/password/change.vue:199]: if you find that you have to make more font sizes, why not just extend bootstrap utility like we're already doing in variables.scss

Comment (PeterThrels) [reply pages/profile/password/change.vue:199]: also use generic class names

**Review by PeterThrels — CHANGES_REQUESTED** [review]

File: assets/scss/_cookie.scss (hunk 1 above)
Comment (LucasSaid005) [reply assets/scss/_cookie.scss:19]: The cookie banner would leave a white space on the right side on small media queries. Now it looks complete. do I revert it?

File: components/organisms/ProfileIdentitySelection.vue (hunk 3 above)
Comment (LucasSaid005) [reply components/organisms/ProfileIdentitySelection.vue:36]: Good Idea. Will look into it now

File: components/molecules/registration/RegistrationCard.vue (hunk 4 above)
Comment (LucasSaid005) [reply components/molecules/registration/RegistrationCard.vue:32]: Looking into it

File: assets/scss/_cookie.scss (hunk 1 above)
Comment (gaucj-t) [reply assets/scss/_cookie.scss:19]: This seems off to me media queries should be avoided. Also i think boostrap 5 has the necessary css classes already for this

File: assets/scss/_nav.scss (hunk 2 above)
Comment (gaucj-t) [reply assets/scss/_nav.scss:21]:
```suggestion
    .navbar, .navbar-light {
```

In order to avoid breaking stuff

File: assets/scss/style.scss [hunk 7]
```diff
@@ -143,3 +143,9 @@ a {
     transition: all 0.2s;
     opacity: 0;
 }
+
+@media (width >= 1400px) {
+    .container {
+        max-width: 1128px;
+    }
+}
```
Comment (gaucj-t) [inline assets/scss/style.scss:151]: why is this ?

File: components/molecules/HeaderComponent.vue (hunk 5 above)
Comment (gaucj-t) [reply components/molecules/HeaderComponent.vue:3]: i believe this is correct from lucas type = light will add navbar-light in behind the scenes. Just double check.

File: assets/scss/_cookie.scss (hunk 1 above)
Comment (PeterThrels) [reply assets/scss/_cookie.scss:19]: if the banner should always be on the right, no need for media query, just me-0 should be good.

File: assets/scss/style.scss (hunk 7 above)
Comment (LucasSaid005) [reply assets/scss/style.scss:151]: ill remove it as it's not required

File: pages/profile/password/change.vue (hunk 6 above)
Comment (LucasSaid005) [reply pages/profile/password/change.vue:199]: Done

File: assets/scss/_nav.scss (hunk 2 above)
Comment (LucasSaid005) [reply assets/scss/_nav.scss:21]: Good idea, implemented it

File: components/molecules/HeaderComponent.vue (hunk 5 above)
Comment (LucasSaid005) [reply components/molecules/HeaderComponent.vue:3]: removed the type. It's looking fine right now. Didn't add .navbar-light to the class too

File: assets/scss/_cookie.scss (hunk 1 above)
Comment (LucasSaid005) [reply assets/scss/_cookie.scss:19]: fixed this issue. edited the right to 0 and made margin in the same file to 16px. looks better now

**Review by gaucj-t — APPROVED** [review]

File: assets/scss/_variables.scss [hunk 8]
```diff
@@ -44,6 +44,9 @@ $spacer: 1rem;
 $card-border-width: 0;
 $card-border-radius: 2rem;
 $dashboard-header-height: 68px;
+$fs-2rem: 2rem;
+$fs-1-75rem: 2rem;
+$fs-2-5rem: 2.5rem;
```
Comment (PeterThrels) [inline assets/scss/_variables.scss:49]: both of them are 2rem ?
can't it be just one of them?

Comment (PeterThrels) [reply assets/scss/_variables.scss:49]: i don't see where it is used

Comment (PeterThrels) [reply assets/scss/_variables.scss:49]: what i meant was to do this

`@import 'bootstrap/scss/utilities';
$utilities: map.merge($utilities, (
  "fs-2rem": (
    property: font-size,
    class: fs-2rem,
    values: (
      2rem: 2rem
    )
  )
));`

this way you can use 'fs-2rem' as a class directly, so no need for <style> tag in component.

**Review by PeterThrels — APPROVED** [review]

File: components/organisms/cards/CardComponent.vue [hunk 9]
```diff
@@ -6,8 +6,10 @@
             class="border-0 cards-card"
             no-body
         >
-            <b-card-body class="card-body onvirtual-card overflow-hidden shadow-hover-sm">
-                <div class="ratio-card overflow-hidden">
+            <b-card-body
+                class="card-body onvirtual-card overflow-hidden shadow-hover-sm ratio ratio-card"
+            >
+                <div>
```
Comment (gaucj-t) [inline components/organisms/cards/CardComponent.vue:12]: do we need this div now ?

**Review by gaucj-t — APPROVED** [review]

File: components/organisms/cards/CardComponent.vue (hunk 9 above)
Comment (LucasSaid005) [reply components/organisms/cards/CardComponent.vue:12]: Yes, tried removing it and the card breaks

**Review by PeterThrels — APPROVED** [review]
